src/background/setup/initBackgroundScript.ts

```diff
@@ -103,6 +103,10 @@ export async function initBackgroundScript() {
 
 			await storeIfUndefined("hintsToggleTabs", new Map());
 			await storeIfUndefined("tabsByRecency", {});
+			await storeIfUndefined(
+				"savedIDsByHost",
+				new Map<string, Map<string, string>>()
+			);
```

Comment:
This wasn't necessary, but this is on me as the previous two are also not necessary.